generator/sbpg/targets/resources/rust/sbp_messages_template.rs

```diff
@@ -140,6 +140,12 @@ impl SbpMessage for (((m.msg_name))) {
     (((m.gps_time_fn)))
 }
 
+impl FriendlyName for (((m.msg_name))) {
+    fn friendly_name() -> &'static str {
+        "(((m.friendly_name)))"
+    }
+}
```

Comment:
This could probably also be just a new function on the `SbpMessage` trait, then you could also use it on the `Sbp` enum without having to manually get a reference to the inner message

Reply:
Oh I see this isn't a method it's an associated function. Might be nice to have `fn friendly_name(&self) -> &'static str` on `SbpMessage`. Otherwise if you want to get the friendly name of a message it would be something like 

```rust
let msg = sbp::iter_messages(r).next().unwrap().unwrap();
match msg {
    Sbp::MsgLog(_) => eprintln!("{}", MsgLog::friendly_name()),
    _ => {}
}
```

vs

```rust
let msg = sbp::iter_messages(r).next().unwrap().unwrap();
eprintln!("{}", msg.friendly_name());
```

Reply:
Having the associated function version also seems useful for some stuff. So maybe leave this as is and then you could add `fn friendly_name(&self) -> &'static str` to the `SbpMessage` trait (which would just call out to `FriendlyName::friendly_name()`)

Reply:
👍